earth_enterprise/src/fusion/autoingest/plugins.src/PacketGen.src

```diff
@@ -353,6 +403,12 @@ void PacketGenAssetVersionImplD::UpdateChildren(
               prevLevelVer.Ref(), /* cached blend */
               cache_alpha_exist ?
                   prevLevelVer.Ref() : std::string() /* cached blend alpha */));
+
+        curTime = std::chrono::high_resolution_clock::now();
+        timeDiff = std::chrono::duration_cast<std::chrono::duration<double>>(curTime - prevTime);
+        notify(NFY_PROGRESS, "UpdateChildren finished cacheRaster block for level %d.  %f seconds elapsed",
+               level, timeDiff.count());
+        prevTime = curTime;
       }
 
```

Comment:
IMO the extra statement wouldn't be too useful because no real processing will take place between the end of the if/else block and a new log statement.  I placed the logs within the if/else to identify which path was taken and get a little more granular output.  We will know how long the if or else blocks took because each branch has a log at the start and finish.

Reply:
Sounds good.